pyproject.toml

```diff
@@ -21,7 +21,7 @@ spyre = "vllm_spyre:register"
 
 [tool.setuptools.packages.find]
 where = ["."]  # list of folders that contain the packages (["."] by default)
-include = ["vllm_spyre"]  # package names should match these glob patterns (["*"] by default)
+include = ["vllm_spyre*"]  # package names should match these glob patterns (["*"] by default)
```

Comment:
what are we gaining from the trailing asterisk?

Reply:
This was the broken part- it was only pulling in files directly in the `vllm_spyre` directory, and nothing nested underneath